### PR TITLE
Users/abmodi/master

### PR DESCRIPTION
**Pull request by abhinitmodi**

### CODE REVIEW

**Comment by pankajkhanzode** [comment]
Signed off.

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 1]
```diff
@@ -43,7 +43,7 @@ function Create-AzureResourceGroup
 
                 foreach($error in $deploymentError)
                 {
-                    Write-Verbose -Verbose $error
+                    Write-Host (Get-LocalizedString -Key "'{0}'" -ArgumentList $error)
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: I hope this doesn't need to update agent resource string, does it?

Comment (NikhilRanjanKumar) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: This error will come from platform, then I don't think we can localize that. If required we can directly print in Host as well

**Comment by NikhilRanjanKumar** [comment]
Signed Off

**Comment by pankagar13** [comment]
merging on behalf of Abhinit modi

**Pull request merged**